aws_lambda.py

```diff
@@ -186,6 +183,80 @@ def validate_changes_collection(event, context):
         raise ValueError("One of the collection did not validate.")
 
 
+def copy_records(event, context):
+    """
+    Note: This lambda is not safe if other user can interact with the destination
+    collection.
+    """
+    source_server_url = event['server']
+    source_auth = os.environ["COPY_RECORDS_AUTH"]
+    source_bucket = os.environ['COPY_RECORDS_SOURCE_BUCKET']
+    source_collection = os.environ['COPY_RECORDS_SOURCE_COLLECTION']
+
+    dest_server_url = os.getenv('COPY_RECORDS_DEST_SERVER', source_server_url)
+    dest_auth = os.getenv("COPY_RECORDS_DEST_AUTH", source_auth)
```

Comment:
Can we do one of:

1. Remove the ability to use two different servers. Anything that needs two different servers could be done using kinto-wizard.
2. Rename `_AUTH` to `_SOURCE_AUTH` for consistency.